interfaces/cfg/keys/doc.go

```diff
@@ -89,6 +89,11 @@ Expected configuration structure is a map, below is a complete example in the YA
 			# 	trace, debug, info, warn, error
 			# Type: string
 			logLevel: "trace"
+			# set the default log level of all the bundled middleware that writes to log and has an error
```

Comment:
I think we need to rephrase it.
This allows to log using a different log level when RPC returned an error....